src/ontodia/diagram/view.ts

```diff
@@ -16,7 +16,7 @@ import {
 } from '../customization/props';
 import { DefaultTypeStyleBundle } from '../customization/defaultTypeStyles';
 import { DefaultLinkTemplateBundle } from '../customization/defaultLinkStyles';
-import { DefaultElementTemplate, DefaultTemplateBundle } from '../customization/templates';
+import { DefaultElementTemplateOwl, DefaultTemplateBundle } from '../customization/templates';
```

Comment:
These changes should be unnecessary (see comments for `stc/ontodia/customization/templates/index.ts`).

Reply:
I do not understand how to satisfy your remarks. I need help